apps/src/code-studio/levels/codeStudioLevels.js

```diff
@@ -144,3 +145,46 @@ export function getContainedLevelResult() {
 export function hasValidContainedLevelResult() {
   return getContainedLevelResult().result.valid;
 }
+
+function getAuthenticityToken() {
+  if (authenticityToken) {
+    return Promise.resolve();
+  } else {
+    return fetch('/user_levels/get_token', {
+      headers: {credentials: 'same-origin'}
+    }).then(response => {
+      if (response.ok) {
+        authenticityToken = response.headers.get('csrf-token');
+      }
+    });
+  }
+}
+
+export function resetContainedLevel() {
+  const levelIds = getLevelIds();
+  if (levelIds.length !== 1) {
+    throw new Error(
+      `Expected exactly one contained level. Got ${levelIds.length}`
+    );
+  }
+  getAuthenticityToken().then(() => {
+    fetch('/delete_predict_level_progress', {
+      method: 'POST',
+      credentials: 'same-origin', // include, *same-origin, omit
```

Comment:
remove stray comment? or help me understand what it means?

Reply:
oh oops, copy pasted comment 🙈